### PR TITLE
Add voice font UI

### PR DESCRIPTION
**Pull request by kbetl-dlb**

### CODE REVIEW

File: android/src/main/kotlin/io/dolby/comms/sdk/flutter/mapper/AudioCaptureModeMapper.kt [hunk 1]
```diff
@@ -43,6 +43,7 @@ class AudioCaptureModeMapper(private val audioCaptureMode: AudioCaptureMode) : M
     companion object {
         fun fromMap(map: Map<String, Any?>): AudioCaptureMode {
             val voiceFont = toVoiceFont(map["voiceFont"] as String?)
+            android.util.Log.d("[KB]", "new voice font: $voiceFont");
```
Comment (graduad) [inline android/src/main/kotlin/io/dolby/comms/sdk/flutter/mapper/AudioCaptureModeMapper.kt:46]: Looks like a temporary log. Should it be removed?

File: test_app/lib/screens/test_buttons/audio_service_test_buttons.dart [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
 import 'package:flutter/material.dart';
+import '../../widgets/voice_font_dialog_content.dart';
```
Comment (graduad) [inline test_app/lib/screens/test_buttons/audio_service_test_buttons.dart:3]: I am not sure about the relative import here. The rest of the widgets seem to be referred by an absolute path.

**Review by graduad — COMMENTED** [review]
I am a bit puzzled that a PR that introduces UI for voice fonts, also changes the SDK.

File: test_app/lib/screens/test_buttons/audio_service_test_buttons.dart (hunk 2 above)
Comment (kbetl-dlb) [reply test_app/lib/screens/test_buttons/audio_service_test_buttons.dart]: changed

File: android/src/main/kotlin/io/dolby/comms/sdk/flutter/mapper/AudioCaptureModeMapper.kt (hunk 1 above)
Comment (kbetl-dlb) [reply android/src/main/kotlin/io/dolby/comms/sdk/flutter/mapper/AudioCaptureModeMapper.kt]: removed

**Pull request merged**